CI/windows/patches/ffmpeg/FFmpeg-devel-avformat-hlsenc-remove-unnecessary-http-https-shutdown-status-operate.diff

```diff
@@ -0,0 +1,14 @@
+diff --git a/libavformat/hlsenc.c b/libavformat/hlsenc.c
```

Comment:
Maybe rename to `.patch` like the other files?

Reply:
Ah, I think the reason this is a .diff is because that's the extension that it came with from the FFmpeg patch system, and I was hesitant/reluctant to change it.  Thoughts?
https://patchwork.ffmpeg.org/project/ffmpeg/patch/20210913021204.22138-1-lq@chinaffmpeg.org/

Reply:
Happened to me as well, I renamed it to patch for consistency with our repos.

Reply:
Oh.  It occurs to me that I'm not actually using that patch here at all, because we're still on FFmpeg 4.2.4 here, where this patch is not needed.  I can probably remove it since it's not OS-specific, and you've already added it to the non-OS-specific patches directory.

Reply:
Removed this.